### PR TITLE
Add Blog Post for Redis Sentinel Webiner - 29 Sept 

### PR DESCRIPTION
**Pull request by Shaad7**
Signed-off-by: Shaad7 <abdullah.alshaad@appscode.com>

### CODE REVIEW

File: content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+On 28th September 2022, Appscode held a webinar on ”Redis Sentinel Ops Requests - Day 2 Lifecycle Management for Redis Sentinel Using KubeDB”. The essential contents of the webinars are:
+- Introducing the concept of Redis Sentinel
+- Deploy and Manage a Cluster in Redis Sentinel Mode
+- Live Demonstration
+- Q & A Session
+
+## Description of the Webinar
+
+Earlier in this webinar, We discussed the `Sentinel` mode in `Redis`. In sentinel mode there are
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md:31]: In sentinel mode, there are two types of instances: redis instances and sentinel instances.

File: content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Take a deep dive into the full webinar below:
+
+<iframe width="800" height="500" src="https://www.youtube.com/embed/LToGVt1-D50" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## What Next?
+
+Please try the latest release and give us your valuable feedback.
+
+* If you want to install **KubeDB**, please follow the installation instruction from [here](https://kubedb.com/docs/v2022.05.24/welcome/).
```
Comment (ArnobKumarSaha) [inline content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md:50]: update to latest version

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

File: content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md (hunk 1 above)
Comment (Emon46) [reply content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md:31]: In sentinel mode, there are two types of Clusters: Redis Database Cluster and Sentinel monitoring Cluster.

File: content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+* If you want to install **KubeVault**, please follow the installation instruction from [here](https://kubevault.com/docs/v2022.06.16/setup/).
```
Comment (Emon46) [inline content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md:52]: KubeVault

File: content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+
+
+## Support
+
+To speak with us, please leave a message on [our website](https://appscode.com/contact/).
+
+To receive product announcements, follow us on [Twitter](https://twitter.com/KubeVault).
+
+If you have found a bug with KubeVault or want to request new features, please [file an issue](https://github.com/kubevault/project/issues/new).
```
Comment (Emon46) [inline content/post/kubedb-redis-sentinel-webinar-2022.09.28/index.md:64]: KubeDB

**Review by Emon46 — CHANGES_REQUESTED** [review]

**Review by Emon46 — APPROVED** [review]
LGTM!

**Pull request merged**